src/structs.h

```diff
@@ -2280,6 +2280,9 @@ typedef struct {
     // set when "arg" points into the last entry of "eval_tofree_ga"
     int		eval_using_cmdline;
 
+    // Is true if we are inside a :class ex command
+    bool	eval_inside_class;
```

Comment:
Instead of introducing a new field in this structure, can you add a new flag to eval_flags?